### ISSUE TITLE
Improve no-aria-label test on SVG

### ISSUE THREAD

**Issue by ffoodd** [ffoodd/a11y.css#146]
Thanks to @Twikito [on Twitter](https://twitter.com/Twikito/status/582564977153875969), this test can be improved:
- SVG may use `aria-labelledby` OR `aria-label` (see [French RGAA3 technical note](http://references.modernisation.gouv.fr/sites/default/files/RGAA3/note_technique.htm#nT1-1));
-- `aria-describedby` is not recommended because of lack of support (see [another technical note in French](http://references.modernisation.gouv.fr/sites/default/files/RGAA3/note_technique.htm#nT1-6));
- thus SVG should have a `role="img"` if non-decorative;
- maybe duplicate criterias about `<img/>` since decorative SVG (and its children) should not have:
 - `aria-label`;
 - `aria-labelledby`;
 - `aria-describedby`;
 - `role` different from `presentation`;
 - `<title>` and `<desc>` empty or absent;
 - no `[title]`.

The existing test can be improved, but we also may look further for some new tests.
